include/h2o.h

```diff
@@ -583,6 +583,10 @@ struct st_h2o_context_t {
      * flag indicating if shutdown has been requested
      */
     int shutdown_requested;
+    /**
+     * index number of the thread that manages this context
+     */
+    size_t thread_index;
```

Comment:
Do we need this?

I am hesitant to start requiring every user of libh2o setting a value just as a result of adding a new optional feature to the status handler.

Reply:
Agreed. Let me think about this a bit more.

Reply:
Removed `thread_index` from the `h2o_context_t` strucutre in d998b1737. Instead the vector is populated based on its size. Caveat is that the array order is no longer stable but I feel that this is ok as a starting point. It is still useful to compare between nodes.

Reply:
Actually this is fine. Once your QUIC work is merged, we're able to tell if the ctx is enabled to handle QUIC or not. If we attach that information to the latency stats, then we're able to group together the QUIC-enabled threads on the dashboard, which would be very nice.